modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/dataSets.spec.ts

```diff
@@ -200,35 +201,55 @@ test('Create parameterized data set via UI', async ({dataSetsPage, page}) => {
 	});
 });
 
-test('Create data set via API', async ({
-	dataSetManagerApiHelpers,
-	dataSetsPage,
-	page,
-}) => {
-	const DEFAULT_DATA_SET_ERC = getRandomString();
-	dataSetERCs.push(DEFAULT_DATA_SET_ERC);
+test(
+	'Assert endpoint with resolved paramater is available as an option',
+	{tag: '@LPD-25230'},
+	async ({dataSetsPage}) => {
+		const cartDataSetConfig = {
+			name: 'Carts',
+			restApplication: '/headless-commerce-delivery-cart/v1.0',
+			restEndpoint:
+				'/v1.0/channels/{channelId}/account/{accountId}/carts',
+			restSchema: 'Cart',
+		};
+
+		const modal = dataSetsPage.newDataSetModal;
+
+		await test.step('Go to Data Sets page and open "New" modal', async () => {
+			await dataSetsPage.goto();
 
-	await test.step('Create Data Set', async () => {
-		await dataSetManagerApiHelpers.createDataSet({
-			...tableSectionsDataSetConfig,
-			erc: DEFAULT_DATA_SET_ERC,
-			label: tableSectionsDataSetConfig.name,
+			await dataSetsPage.newDataSetButton.click();
+
+			await expect(modal.nameInput).toBeVisible();
 		});
-	});
 
-	await test.step('Navigate to Data Sets page', async () => {
-		await dataSetsPage.goto();
-	});
+		await test.step('Assert endpoint with resolved paramater is available', async () => {
+			await modal.restApplicationField.click();
 
-	await assertTableColumnLabels(page);
+			await modal.restApplicationOptions
+				.getByRole('option', {name: cartDataSetConfig.restApplication})
+				.click();
 
-	await assertTableCellContent({
-		dataSetConfig: tableSectionsDataSetConfig,
-		page,
-	});
+			await expect(modal.restSchemaField).toBeVisible();
 
-	await assertTableActionLabels(page);
-});
+			await modal.restSchemaField.click();
+
+			await modal.restSchemaOptions
+				.getByRole('option', {name: cartDataSetConfig.restSchema})
+				.click();
+
+			await expect(modal.restEndpointField).toBeVisible();
+
+			await modal.restEndpointField.click();
+
+			await expect(
+				modal.restEndpointOptions.getByRole('option', {
+					name: cartDataSetConfig.restEndpoint,
+				})
+			).toBeVisible();
```

Comment:
@gianmarcobrunialti  My initial thinking in https://github.com/liferay-frontend/liferay-portal/pull/4367#pullrequestreview-2230350122 was to have a test in `data-set-fragment`, testing sample fragment. But unfortunately, we cannot do this. Resolver added in this PR only works with commerce context:
```java
CommerceContext commerceContext =
	(CommerceContext)httpServletRequest.getAttribute(
		CommerceWebKeys.COMMERCE_CONTEXT);
```

We will need to add that test later on, on the real commerce implementation.